### ISSUE TITLE
async_setup_platforms depreciated in HA 2023.3

### ISSUE THREAD

**Issue by rcblackwell** [jrfernandes/ontario_energy_board#21]
Noted a warning in my HA logs regarding depreciation of async_setup_platforms failing in HA release 2023.3 onward. Thought it should be brought to attention in case it hasn't been noted yet.
```
Logger: homeassistant.helpers.frame
Source: helpers/frame.py:77
First occurred: 2:37:09 PM (1 occurrences)
Last logged: 2:37:09 PM

Detected integration that called async_setup_platforms instead of awaiting async_forward_entry_setups; this will fail in version 2023.3. Please report issue to the custom integration author for ontario_energy_board using this method at custom_components/ontario_energy_board/__init__.py, line 21: hass.config_entries.async_setup_platforms(entry, PLATFORMS)
```

**Comment by jprasm** [comment]
Same issue here.

**Comment by jprasm** [comment]
Any updates here? Error persists in 2023.4x

**Comment by magnonan** [comment]
Same here...any updates?

**Comment by jrfernandes** [comment]
This is now fixed with [v0.3.3](https://github.com/jrfernandes/ontario_energy_board/releases/tag/v0.3.3)
